### DISCUSSION TITLE
Using inside Clojure

### DISCUSSION THREAD

**Discussion by prepor** [clavascript/clavascript#236] [Q&A]
Hello!

I'm trying to figure out the current possible workflow to use squint in Clojure as a library. It publishes only NPM, no jars. And I can't use it via deps.edn's git support because the project contains submodules.

Thank you!

**Comment by prepor** [comment]
In fact, maybe I'm wrong that tools.deps doesn't support git submodules? But that's what I get if add `io.github.squint-cljs/squint {:git/sha "9c54c6ea5bec5ddd7f4a710ee08ed5158e236a10"}` to deps.edn:

```
Error building classpath. Manifest type not detected when finding deps for io.github.squint-cljs/compiler-common in coordinate #:local{:root "/Users/prepor/.gitlibs/libs/io.github.squint-cljs/squint/9c54c6ea5bec5ddd7f4a710ee08ed5158e236a10/compiler-common"}
```

**Reply by borkdude** [reply]
Yes, this is a problem. I might have to find a way to deal with a common code base in two projects. Perhaps I'll have to move squint and cherry within the same git repo, which also isn't optimal imo.

You could try commit https://github.com/squint-cljs/squint/tree/1100357f8f12795f1e636caa8b9a8134cd351b46 which didn't have the submodule yet.

**Reply by borkdude** [reply]
I now removed the submodule again, so you should be able to use it as a normal git dep again
